en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -0,0 +1,52 @@
+# Multi-Vehicle Simulation with JMAVSim
+
+This topic explains how to simulate multiple UAV (multicopter) vehicles using JMAVSim and SITL.
+
+> **Tip** This is the easiest way to simulate multiple vehicles running PX4. 
+  It works on PX4 v1.8.0 and later.
+
+## Instances on Separate Ports
+
+To start multiple instances on separate ports:
+
+1. Build PX4
+   ```
+   make posix_sitl_default
+   ```
+1. Run **sitl_multiple_run.sh**, specifying the number of instances to start (e.g. 2):
+   ```
+   ./Tools/sitl_multiple_run.sh 2
+   ```
+1. Start the first instance:
+   ```
+   ./Tools/jmavsim_run.sh
```

Comment:
@bkueng According to http://dev.px4.io/en/simulation/#default-px4-mavlink-udp-ports port 14560 is used for connecting PX4 to simulators - ie the SITL / jMAVSim connection. 

Just to confirm:
1. Is the default port (for first instance) 14560?
2. When you set to 14561 is this the simulator port, or the port to which you connect a GCS?

Reply:
So what does a GCS connect on then? ie if you set port 14561 , does QGC try to connect to 14541 or something else?

Reply:
All run on 14540 as usual.

Reply:
Thank you. FYI I updated doc with this information in https://github.com/PX4/Devguide/commit/360a30ecba13069349fbba97bfc2b1058e8da44f